packages/server/src/documents/documents.controller.ts

```diff
@@ -66,7 +66,7 @@ export class DocumentsController {
       commitResult.header.job_id,
       user.account,
     );
-    return await this.databaseService.documents.updateById(document._id, {
+    return await this.databaseService.documents.update({'header.document_id': document.header.document_id}, {
```

Comment:
I am wondering if we can get this information from the document. When we commit does the node do any mutation on the doc, add a prop or something?

Reply:
we are getting this information from the document

Reply:
I am thinking that we can use the webhook in the future because can be error prone. We can not be 100% that the other user received the doc in is in the database

Reply:
but i think we will need to update the webhook code rn bc it updates the single document of the user, but the documents would actually be different atm due to the differing organizationIds and we would need to update both documents